napalm_eos/eos.py

```diff
@@ -985,16 +985,20 @@ def get_mac_address_table(self):
     def get_route_to(self, destination='', protocol=''):
         routes = dict()
 
+        if protocol.lower() not in ['', 'bgp', 'connected', 'isis', 'ospf', 'rip', 'static']:
```

Comment:
Can you add in that list also ```direct``` and when user requests ```direct``` to rewrite the value of the ```protocol``` var to ```connected```?